xbmc/GUIInfoManager.cpp

```diff
@@ -188,6 +188,18 @@ typedef struct
   int  val;
 } infomap;
 
+const infomap string_bools[] =   {{ "isempty",          STRING_IS_EMPTY },
+                                  { "equals",           STRING_EQUALS },
+                                  { "startswith",       STRING_STARTS_WITH },
+                                  { "endswith",         STRING_ENDS_WITH },
+                                  { "contains",         STRING_CONTAINS }};
+
+const infomap integer_bools[] =  {{ "equals",           INTEGER_EQUALS },
+                                  { "greater",          INTEGER_GREATER_THAN },
+                                  { "greaterorequals",  INTEGER_GREATER_OR_EQUALS },
+                                  { "less",             INTEGER_LESS_THAN },
+                                  { "lessorequals",     INTEGER_LESS_OR_EQUALS }};
```

Comment:
> They are very verbose. Perhaps use the de facto standard eq, gt, ge, lt, le notation?

I don't want to use short-hands because it's less readable and does not be the same as for Strings e.g. `String.Equals()`

> Also, why not 'not equal`?

you are able to negate the expression by adding `!` in front
```
!Integer.Equals(argument,1)
!String.Contains(str,subtring)
```